content/_data/changelogs/lts.yml

```
@@ -4389,6 +4389,331 @@
+        title: Jetty 9.4.33.v20201020 changelog
+    message: |-
+      Winstone 5.12: Update Jetty from 9.4.30.v20200611 to 9.4.33.v20201020.
+      Introduce LowResourceMonitor from Jetty by upgrading to Winstone 5.12.
+  - type: rfe
+    category: rfe
+    pull: 5035
+    issue: 64039
+    authors:
+    - jtnord
+    message: |-
+      Improve performance of authorisation strategies when the authentication realm is case insensitive.
+
+  # pull: 5033 (PR title: JDK 11.0.9 / 8.0.272+ compatible FunctionsTest#printThrowable())
+  # pull: 5064 (PR title: Update the pinned version of ATH in use to a more recent version)
+
+  lts_changes: # compared to lts_predecessor 2.249.3 (selected by personal review)
+
+  - type: major bug
+    category: major bug
+    authors:
+    - tam116
+    references:
+      - issue: 64212
+      - url: "https://github.com/jenkinsci/packaging/pull/198"
+        title: pull 198
+    message: |-
+      Fix Debian / Ubuntu Java version check for Java 11.0.9.1.
+
+  - type: major rfe
+    category: major rfe
+    pull: 4909
+    references:
+      - pull: 4909
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/224/README.adoc
+        title: JEP-224
+    authors:
+    - timja
+    message: |-
+      Graduate Overall/SystemRead permission to general availability (GA) status.
+
+  - type: major rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Remove deprecated key exchange and MAC algorithms.
+    references:
+      - pull: 4951
+      - url: "https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md#27"
+        title: SSHD module 2.7 changelog
+
+  - type: rfe
+    category: rfe
+    pull: 4951
+    authors:
+    - daniel-beck
+    message: >
+      SSHD module 2.7: Allow configuring disabled key exchange and MAC algorithms through system properties.
+    references:
+      - pull: 4951
+      - url: "https://github.com/jenkinsci/sshd-module/blob/master/CHANGELOG.md#27"
+        title: SSHD module 2.7 changelog
+
+  - type: rfe
+    category: rfe
+    pull: 4926
+    issue: 63592
+    authors:
+    - fqueiruga
+    message: |-
+      Improve the layout and clarity of the page displayed when jobs are not yet created.
+
+  - type: rfe
+    category: rfe
+    pull: 4920
+    issue: 60972
+    authors:
+    - romenrg
+    message: |-
+      Name the first breadcrumb "Dashboard" for clarity.
+
+  - type: rfe
+    category: rfe
+    pull: 5016
+    authors:
+    - daniel-beck
+    message: |-
+      Remove TFS Plugin suggestion from Setup Wizard.
+
+  - type: rfe
+    category: rfe
+    pull: 4952
+    authors:
+    - daniel-beck
+    message: |-
+      Disable autocomplete of username on login form.
+
+  - type: rfe
+    category: rfe
+    pull: 4950
+    issue: 63795
+    authors:
+    - aHenryJard
+    message: |-
+      Allow users with the <code>Jenkins.MANAGE</code> permission to restart and safe restart Jenkins.
```

```suggestion
      Allow users with the Jenkins/Manage permission to restart and safe restart Jenkins.
```

Code references in the user changelog should be an absolutely exception.

---

Fixed in a6adc4203a51928bc2e3f233f4d041baa078bd35